LEGO1/lego/legoomni/include/legoscripts.h

```diff
@@ -0,0 +1,32 @@
+#ifndef LEGOSCRIPTS_H
+#define LEGOSCRIPTS_H
+
+typedef enum IntroScript {
+	IntroScript_Lego_Movie = 0,
+	IntroScript_Mindscape_Movie,
+	IntroScript_Intro_Movie,
+	IntroScript_Outro_Movie,
+	IntroScript_BadEnd_Movie,
+	IntroScript_GoodEnd_Movie
+};
+
+typedef enum InfomainScript {
+	InfoMainScript_WelcomeDialogue = 500,
+	InfoMainScript_RandomDialogue1 = 502,
+	InfoMainScript_LetsGetStarted = 504,
+	InfoMainScript_ReturnBack = 514,
+	InfoMainScript_ExitConfirmation = 522,
+	InfoMainScript_GoodEndingDialogue = 539,
+	InfoMainScript_BadEndingDialogue = 540,
+	InfoMainScript_PepperCharacterSelect = 541,
+	InfoMainScript_MamaCharacterSelect = 542,
+	InfoMainScript_PapaCharacterSelect = 543,
+	InfoMainScript_OfficierCharacterSelect = 544,
+	InfoMainScript_LoraCharacterSelect = 545,
+};
+
+typedef enum SndAmimScript {
+	SndAmimScript_BookWig = 400
+};
```

Comment:
Small styling nitpick - should these be UPPERCASE like other enums; such as `TransitionType` in MxTransitionManager?

Reply:
Also in c++, you don't need to add the typedef.

Reply:
My personal opinion is UPPERCASE should only be used for constants. (That is, where the value has some meaning other than just uniquely identifying a set member.)  The style guide says these should be like `e_nameOfEnumMember`, but I think this is clear enough.

Reply:
Didn't realize we had one in the style guide - that's my bad. It's good to get this all sorted out though.